camelot/view/controls/editors/customeditor.py

```diff
@@ -180,3 +180,38 @@ def sizeHint(self):
             size_hint.setWidth(max(size_hint.width(), self.size_hint_width))
         return size_hint
 
+    def get_validator(self, validator_type, validator_state):
+        from vfinance.view import validators
+        if validator_type is None:
+            return None
+        if validator_type == validators.BankingNumberValidator.__name__:
+            return validators.BankingNumberValidator()
```

Comment:
Hier kunnen we natuurlijk ook een bstaande instance gebruiken...

Reply:
ja, in principe zou je de validator op de editor kunnen zetten in de constructor

en dan enkel nog de validator type veranderen

zie maar wat u het eenvoudigste lijkt.  op zich is dit hier 'waste of validators', maar dat is de huidige code ook al, so it doesn't really mater ...